index.html

```diff
@@ -242,21 +242,14 @@ <h2 class="contact-h2">Contact me</h2>
                     that you need built or a project that needs coding. I’d love to help with it</p>
             </div>
             <article class="form-article">
-                <form action="https://formspree.io/f/xpzcccob" method="POST" class="contact-form" id="contact-form">
-                    <label for="name"></label>
-                    <input type="text" name="Name" placeholder="Write name." required maxlength="30" id="name">
-                    <label for="email"></label>
-                    <input type="email" name="Email" placeholder="example@gmail.com" required id="email">
-                    <label for="message-container"></label>
+                <form action="https://formspree.io/f/mknagrgk" method="POST" class="contact-form">
+                    <input type="text" name="Name" placeholder="Write name." required maxlength="30">
+                    <input type="email" name="Email" placeholder="example@gmail.com" required>
                     <textarea name="message" id="message-box" cols="30" rows="10"
                         placeholder="write your message in here" maxlength="500" required></textarea>
-                    <button type="submit" class="get-in-touch-btn
-                    ">Get It Touch</button>
-                    <div class="error-message">
-                        <small id="error"></small>
-                    </div>
+                    <button type="submit">Get It Touch</button>
                 </form>
-                <img src="/images/contact-ba-bar.png" alt="bar at the bottom" class="home-indicator">
+                <img src="./images/contact-ba-bar.png" alt="bar at the End">
             </article>
```

Comment:
- This section of your portfolio does not exactly look like what is in the design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏
Your project:
-- The background images are missing for this section. Please fix this 😄 
<img width="147" alt="ethethte" src="https://user-images.githubusercontent.com/73607512/213175081-7a0102c1-fbd1-4b92-bbf1-b5d24b16ab94.PNG">
-- Your submit button seems to be misaligned on bigger screens, please fix this 🙏 
<img width="340" alt="ryskyk" src="https://user-images.githubusercontent.com/73607512/213176968-5f7be9e5-b448-4f6d-b68f-5a7fb9913dd5.PNG">